### PR TITLE
Take fill-alpha into account with default icons for FileAttachment annotations (issue 16800)

### LINKED ISSUE #16800: How do I remove embedded images
Attach (recommended) or Link to PDF file here:

[TS354 Week 2 slides.pdf](https://github.com/mozilla/pdf.js/files/12287878/TS354.Week.2.slides.pdf)

Configuration:
- Web browser and its version: Chrome 115.0.5790.171
- Operating system and its version: Windows 11 home 22H2
- PDF.js version: 3.9.49
- Is a browser extension: yes

Steps to reproduce the problem:
1. Open pdf
2. Go to page 2

What is the expected behavior? (add screenshot)
Not see the pin

What went wrong? (add screenshot)
I'd just like to just see the image underneath the black pin

image

### PR DESCRIPTION
**Pull request by Snuffleupagus**

### CODE REVIEW

File: src/core/annotation.js [hunk 1]
```diff
@@ -4652,6 +4652,9 @@ class FileAttachmentAnnotation extends MarkupAnnotation {
     const name = dict.get("Name");
     this.data.name =
       name instanceof Name ? stringToPDFString(name.name) : "PushPin";
+
+    const fillAlpha = dict.get("ca");
+    this.data.fillAlpha = typeof fillAlpha === "number" ? fillAlpha : null;
```
Comment (calixteman) [inline src/core/annotation.js:4657]: nit: maybe you should validate or clamp the value.

**Review by calixteman — APPROVED** [review]
LGTM. Thank you.

File: src/core/annotation.js (hunk 1 above)
Comment (Snuffleupagus) [reply src/core/annotation.js]: Sure; I went with adding more validation, since that seems overall safer.

**Pull request merged**